modules/caddyhttp/templates/tplcontext.go

```diff
@@ -183,20 +183,28 @@ func (c TemplateContext) funcHTTPInclude(uri string) (string, error) {
 	buf.Reset()
 	defer bufPool.Put(buf)
 
+	// Create a new virtual request
```

Comment:
This comment is redundant, it's already explained elsewhere and it restates the variable name
```suggestion
```